modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -412,6 +412,31 @@ def test_loc(data):
     with pytest.raises(KeyError):
         modin_df.loc["NO_EXIST"]
 
+    # From issue #3764
+    pandas_df = pandas.DataFrame([[1, 2, 3], [4, 5, 6]])
+    modin_df = pd.DataFrame([[1, 2, 3], [4, 5, 6]])
+
+    for left, right in [(2, 1), (6, 1), (lambda df: 70, 1), (90, 70)]:
+
+        def _test_loc_rows(df):
+            df.loc[left] = df.loc[right]
+            return df
+
+        eval_general(modin_df, pandas_df, _test_loc_rows)
+
+    for columns in [10, (100, 102), (2, 6), [10, 11, 12], "a", ["b", "c", "d"]]:
+        if isinstance(columns, tuple) and len(columns) == 2:
+
+            def _test_loc_cols(df):
+                df.loc[:, columns[0] : columns[1]] = 1
+
+        else:
+
+            def _test_loc_cols(df):
```

Comment:
ditto about moving funciton definitions out of the loop

Reply:
Done.